_ja/functions/andThen.md

```diff
@@ -0,0 +1,12 @@
+---
+name: andThen
+---
+
+# `andThen`
+
+@include [signatures/andThen.md]
+
+`andThen` は匿名関数を作成します。その関数は、受け取った値にまずこの関数を適用し、次にその値に `g` を適用した結果を返します。
+これは [`compose`]({{ site.baseurl }}{% link _ja/functions/compose.md %}) の逆です。
```

Comment:
This is definitely a thing to be implemented (as discussed in #4) but until a proper way to express relationships between functions is decided I'd like to preserve the same documentation across translations.

Also, there is no need to use Liquid tags when the same effect can be achieved using the Kramdown syntax:

    [`compose`](./compose)

Reply:
Oops, I forgot to remove these changes since not in other translation.

I will separate these changes in subsequent PR with updates in English/Spanish.

Reply:
I thank you your effort, but I prefer to separate concerns in different PRs.
Let's translate what is currently documented in the main language (English) and then let's think about the best way to add relationships between functions.

Reply:
Removed additional paragraphs in https://github.com/superruzafa/visual-scala-reference/pull/11/commits/f499d0cc9cf329e79aaf92b2e630138c1238d1dd